services/workflows-service/src/workflow/workflow.controller.external.ts

```diff
@@ -221,7 +221,7 @@ export class WorkflowControllerExternal {
     @ProjectIds() projectIds: TProjectIds,
     @CurrentProject() currentProjectId: TProjectId,
   ): Promise<void> {
-    return await this.service.event(
+    await this.service.event(
```

Comment:
Curious about this change

Reply:
send event now return updated runtime after the event was sent